packages/web/src/scss/tools/_form-fields.scss

```diff
@@ -139,7 +139,7 @@
 
         @if $field-name != 'CheckboxField' or $field-name != 'RadioField' {
             :is(.#{$field-name}--#{$validation-state-name}, .#{$field-name}.has-#{$validation-state-name})
-                :is(.#{$field-name}__message, [data-element='validator_message']) {
+                :is(.#{$field-name}__message, [data-element='validation-text']) {
```

Comment:
isn't this BC?

Reply:
I think not. I think that this is unused. But let me know if I should consider this opposite.

Reply:
Ok, it is a BC. Going back.